src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,16 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import './TodoInfo.scss';
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo: { userId, completed, title, user } }) => (
+  <article
+    className={cn('TodoInfo', { 'TodoInfo--completed': completed })}
+  >
+    <h2 className="TodoInfo__title">{title}</h2>
+
+    {userId && user && (
```

Comment:
```suggestion
    {user && (
```

there's no need to check wether `userId` exists. It always exists)